bundle/edu.gemini.pit/src/main/scala/edu/gemini/pit/ui/editor/CoiEditor.scala

```diff
@@ -14,10 +14,14 @@ import edu.gemini.pit.ui.util._
 import swing.event.ValueChanged
 
 object CoiEditor {
-  def open(coi: CoInvestigator, editable:Boolean, parent: UIElement) = new CoiEditor(coi, editable).open(parent)
+  def open(coi: CoInvestigator, editable: Boolean, parent: UIElement, setup: CoiEditor => Unit = _ => {}) = {
```

Comment:
It should still be returning `Option[CoInvestigator]`, as that is what is returned by the `open` method on `CoiEditor` instances. The `setup` function simply allows configuration of the `CoiEditor` UI so that we can do things like set the focus on certain UI elements, e.g. the `Phone` text field as was requested by this REL task.

I do think it would be helpful to make the return type explicit, so I am going to add that to the method signature for both `CoiEditor.open` and `PIEditor.open` (which, of course, returns `Option[PrincipalInvestigator]`.